src/extractors/custom/www.rbbtoday.com/index.js

```diff
@@ -0,0 +1,34 @@
+export const WwwRbbtodayComExtractor = {
+  domain: 'www.rbbtoday.com',
+
+  title: {
+    selectors: ['h1'],
+  },
+
+  author: {
+    selectors: ['.writer.writer-name'],
+  },
+
+  date_published: {
+    selectors: [['header time', 'datetime']],
+  },
+
+  dek: null,
+
+  lead_image_url: {
+    selectors: [['meta[name="og:image"]', 'value']],
+  },
+
+  content: {
+    selectors: ['.arti-content'],
+
+    transforms: {
+      '.giga-list': $node => {
```

Comment:
I feel like this element should actually be "cleaned" out, rather than transformed.
Correct me if I'm wrong, but this section contains links to similar/related/other articles under the same main category/tag of the current article, so they're not exactly part of the core article content.

Reply:
I feel that related links are also included in the core article, but in that case I would like to clean `.arti-giga`.